packages/core/src/common/catalog-entities/kubernetes-cluster.ts

```diff
@@ -127,7 +127,13 @@ export class KubernetesCluster<
         context.menuItems.push({
           title: "Disconnect",
           icon: "link_off",
-          onClick: () => requestClusterDisconnection(this.getId()),
+          onClick: () => {
+            requestClusterDisconnection(this.getId());
+            broadcastMessage(
+              IpcRendererNavigationEvents.NAVIGATE_IN_APP,
+              "/catalog",
+            );
```

Comment:
Thinking aloud, could we instead say "NAVIGATE_BACK_IN_APP"? Or is that not currently supported?

If not, the LGTM